scriptmodules/ports/hexen2.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hexen2"
+rp_module_desc="Hexen II - Hammer of Thyrion source port"
+rp_module_licence="GLP2"
+rp_module_help="For registered version, please add your full version PAK files to $romdir/ports/hexen2/data1/ to play. These files for the registered version are required: pak0.pak, pak1.pak and strings.txt. The registered pak files must be patched to 1.11 for Hammer of Thyrion."
+rp_module_section="opt"
+rp_module_flags=""
+
+
+# Get depends required by hexen2
+function depends_hexen2() {
+    getDepends cmake libsdl1.2debian libsdl-image1.2 libsdl-mixer1.2 libsdl-mixer1.2-dev libsdl-net1.2 libsdl-net1.2-dev timidity
```

Comment:
only -dev packages should be included. the binary packages are depends of them.

Reply:
Would that be only these files then?
libsdl1.2-dev
libsdl-image1.2-dev
libsdl-mixer1.2-dev
libsdl-net1.2-dev

Does this -dev packages only apply to sdl libs packages?

Reply:
it applies to all libraries.

Reply:
This still needs sorting in your latest changes.

Reply:
The libsdl packages has now been changed to dev packages in the latest commit. Is there anything left to do?

The script seems ready to merge. :-)